crates/chainio/clients/fireblocks/src/client.rs

```diff
@@ -1,11 +1,29 @@
+use crate::error::FireBlockError;
+use chrono::Utc;
+use jsonwebtoken::{encode, Algorithm, EncodingKey, Header};
+use mime::APPLICATION_JSON;
+use once_cell::sync::Lazy;
+use reqwest::header::{HeaderMap, HeaderValue, AUTHORIZATION, CONTENT_TYPE};
+use serde::{Deserialize, Serialize};
+use sha2::{Digest, Sha256};
 use std::collections::HashMap;
+use uuid::Uuid;
 
-use serde::{Deserialize, Serialize};
+const X_API_KEY: &str = "X-API-KEY";
```

Comment:
@dralves  added the api key here as a constant

Reply:
thanks, there's still an instance not using the constant, though. but more importantly that comment was more about making sure that every comment is either addressed, of if it can't or won't be addressed that the comment is replied to and left unresolved for the reviewer to take a look. hope that makes sense